### ISSUE TITLE
Have newsroom display medium posts

### ISSUE THREAD

**Issue by HeyItsFelipe** [inland-empire-software-development/main#184]
-Create a IESD medium account
-Copy and paste current newsroom post into IESD medium account
-Replace current posts in newsroom with medium posts

**Comment by tonynguyen111997** [comment]
@HeyItsFelipe I'm still currently working on this. Just for clarification, for "Replace the current posts in newsroom with medium posts" are we looking to replace where the source of information is coming from? For example, right now it's coming from our own API, but for this task are we looking to have it come from Medium? 

This looks like it provides a JSON of our medium posts: https://api.rss2json.com/v1/api.json?rss_url=https://medium.com/feed/@community_59767

**Comment by HeyItsFelipe** [comment]
@tonynguyen111997 Yes, that is correct.  The source should change from our own API to the IESD Medium.  That JSON of our medium posts could be used to display the articles on the IESD page.
